### DISCUSSION TITLE
为什么官方文档的隐藏调试专用，只隐藏了codeSandBox，而没有隐藏codePen和Stackblitz，有什么区别吗

### DISCUSSION THREAD

**Discussion by barKhan92** [ant-design/ant-design#49123] [General]
为什么官方文档的隐藏调试专用，只隐藏了codeSandBox，而没有隐藏codePen和Stackblitz，有什么区别吗

**Comment by zombieJ** [comment]
因为 CodeSandbox 开始收费了，它把默认 Draft 都变成 private 的。导致很多人开 issue 都没有注意到要设置为公开的，以至于提供的都是空的重现。

**Comment by crazyair** [comment]
只是不推荐用 codesandbox，所以找了个方式隐藏掉了
